karma.conf.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+var util = require('util');
+var pkg = require('./package.json');
+var extend = util._extend;
+var geSaLaKaCuLa = require('gesalakacula');
+
+// No Karma options are passed after the double dash option (`--`)
+// Example : karma start --single-run -- --polyfill
+//        >> { _: [], polyfill: true }
+
+var _argv = process.argv;
+var argv = require('minimist')(_argv.slice(_argv.indexOf('--') + 1));
+
+
+var options = extend({
+  travis: process.env.TRAVIS,
+  polyfill: false,
+  saucelabs: false
+}, argv);
+
+////
+
+module.exports = function (config) {
+
+
+  // Default Config
+  config.set({
+    basePath: '',
+    frameworks: ['mocha', 'expect'],
+    files: [
+      'node_modules/traceur/bin/traceur.js',
+
+      'dist/es6-module-loader' +
+      (options.polyfill ? '' : '-sans-promises')
+      + '.src.js',
+
+      'test/_browser.js',
+      'test/custom-loader.js',
+      'test/*.spec.js',
+
+      {pattern: 'test/{loader,loads,syntax,worker}/**/*', included: false},
+      {pattern: 'node_modules/when/es6-shim/Promise.js', included: false},
+      {pattern: 'dist/es6-module-loader.js', included: false}
+    ],
+    reporters: ['mocha'],
+    browsers: ['Chrome', 'Firefox']
+  });
+
+  if (options.travis) {
+    // TRAVIS config overwrite
+    config.set({
+      singleRun: true,
+      reporters: ['dots'],
+      customLaunchers: {
+        'TR_Chrome': {
+          base: 'Chrome',
+          flags: ['--no-sandbox']
+        }
+      },
+      browsers: ['TR_Chrome', 'Firefox']
+    });
+  }
+
+  if (options.saucelabs) {
+    var customLaunchers = geSaLaKaCuLa({
+      'Windows 7': {
+        'internet explorer': '9..11'
```

Comment:
Aaah! I was struggling finding out test by test, which one can pass without Traceur...
was planing to extract them somehow...
it's good to know what I'm actually suppose to test on IE8 
Thanks :+1: